test/e2e/framework/framework.go

```diff
@@ -49,6 +52,12 @@ func createLighthouseClient(restConfig *rest.Config) *lighthouseClientset.Client
 	return clientSet
 }
 
+func setGlobalnetEnabled() {
+	listOptions := metav1.ListOptions{LabelSelector: "app=submariner-globalnet"}
+	podList, _ := framework.KubeClients[framework.ClusterB].CoreV1().Pods(framework.TestContext.SubmarinerNamespace).List(listOptions)
+	framework.TestContext.GlobalnetEnabled = podList != nil && len(podList.Items) > 0
+}
```

Comment:
Do we still need this now that we're passing the CLI flag?

Reply:
> Do we still need this now that we're passing the CLI flag?

No, this will be cleaned up. Testing helm locally, just wanted to push to see if any operator specific issues.